agent/01_agent_requirements.sh

```diff
@@ -66,8 +66,5 @@ if [[ "${AGENT_E2E_TEST_BOOT_MODE}" == "ISO_NO_REGISTRY" ]]; then
     
    early_deploy_validation
 
-   # Extract an updated client tools from the release image
-   extract_oc "${OPENSHIFT_RELEASE_IMAGE}"
```

Comment:
It does not seem correct to me, since an initial version of `oc` is downloaded in the very first step https://github.com/openshift-metal3/dev-scripts/blob/29bb68793c0825229d9757c135fa090ab4572436/01_install_requirements.sh#L172

Reply:
Ah, I see. I think the 01_agent_requirements should be used only for basic installations of dependency strictly required by ABI (and probably also the iso builder checkout could be moved later).

Btw, could you please open a proper JIRA task for this issue (AGENT is ok)? Otherwise it could become diffcult to track all the changes, ie NO-ISSUE IMHO should be used only in very rare cases, thanks.

Reply:
Whats the next best target for checking out the iso builder code? I don't want to creare a new target which only clones the code. Can't be embeded in 03_build_installer.sh either.

Reply:
IMHO it's in `06_agent_create_cluster.sh` step. This is the step where we usually create the ISO to be attached to the VMs. In this case we're simply following a different approach, but still the goal it's to generate an ISO

Reply:
Done